.editorconfig

```diff
@@ -1,8 +1,14 @@
 root = true
 
+# default
 [*]
-end_of_line = lf
-insert_final_newline = true
 charset = utf-8
-indent_style = space
+end_of_line = lf
 indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
```

Comment:
I took the freedom to sort the lines above ☝🏽
The real addition in this is PR is the `trim_trailing_whitespace` config

_default_: true
for `*.md` files: false